packages/server/graphql/mutations/addMissingJiraField.ts

```diff
@@ -87,12 +87,19 @@ const addMissingJiraField = {
     }
     const {fieldType, fieldId} = dimensionField
 
-    const screensResponse = await manager.getScreens(cloudId)
+    console.log(`Adding missing Jira field, fieldId: ${fieldId}, cloudId: ${cloudId}, projectKey: ${projectKey}`)
```

Comment:
@mattkrick I was thinking about keeping these logs forever. 

* This fix field functionality most likely would be called once or twice per Jira project, which shouldn't be very often.
* In this case, the function did not find the right project and failed with timeout. Just logging errors alone would not give enough information why the function did not fix the field.
* It feels that functions that perform a complex sequence of actions and call third-party API on the backend should record their actions. Suppose we changed something very wrong on the customer third-party service, without logs it will be hard to understand what caused such a result. It is also hard to reproduce, since we can't easily test on same data. I think there should be a policy about what we should log and what we shouldn't.

What do you think?

Reply:
Maybe we should create a separate issue to discuss logging policy and then add it in PR review guide?

Reply:
a fantastic discussion topic! i think we only want to log things when there's an error, otherwise we'll end up with so many logs being printed that it becomes impossible to find the ones you're looking for. in a future where our app makes 1,000,000 calls to jira & only 1 of them is an error, we'll want to make sure that 1 stands out real clearly in the logs. it's also cheaper that way since datadog & sentry both charge more when we send them more data.

Reply:
Created an issue for removing logs #6689